README.md

```diff
@@ -1,6 +1,4 @@
-<img src="notebooks/images/NCAR_CISL_NSF_banner.jpeg" alt="NCAR CISL logo" />
-
-# NCAR Dask Tutorial
+# Dask Cookbook
```

Comment:
Should we change the name to Introductions to Dask?

Reply:
There is already a subsection called Introductions to Dask, but that might be a little confusing, but no strong opinion.

Reply:
On the second thought, I think the "Dask Cookbook" alone might be better.